tests/_data/models.py

```diff
@@ -1310,6 +1310,28 @@ def get_bom_with_definitions_standards() -> Bom:
     )
 
 
+def get_bom_v1_6_with_provides() -> Bom:
```

Comment:
please rename to `get_bom_with_provides`.

----

there is no intention to have models for certain CDX versions only.
In fact, it is intended to test the serialization with a target that is expected to omit certain parets.